lib/service/src/main/java/io/atlasmap/service/AtlasService.java

```diff
@@ -410,6 +410,7 @@ public Response deleteMappingProjectById(
             LOG.warn("{} is not a directory - removing anyway", mappingFolderFile.getAbsolutePath());
         }
         AtlasUtil.deleteDirectory(mappingFolderFile);
+        admHandlerMap.clear();
```

Comment:
This should be `admHandlerMap.remove(mappingDefinitionId)`, as `clear()` removes handlers for all existing `mappingDefinitionId`.

Reply:
Also can you add a test case to verify the mapping project is removed?

Reply:
Also can you add a test case to verify the mapping project is removed?